### ISSUE TITLE
How to add new camera to Meshroom

### ISSUE THREAD

**Issue by ostepan2006** [alicevision/meshroom#1452]
**Describe the problem**
Hello. 
How can I add a new camera to the Meshroom to do 3D reconstruction from her images?
Where shouid I set intrinsic calibration parameters of the camera and some other parameters for example sensor sizes?

Could you give me comprehension tutorial?

Thanks

**Comment by natowi** [comment]
https://github.com/alicevision/meshroom/wiki/Add-Camera-to-database
and
https://github.com/alicevision/meshroom/issues/921#issuecomment-637569290
(needs to be added to the doc)

**Comment by pchambre** [comment]
Hello, 

I am trying to model a small part. To reduce non-object parts of my image, I took my photos with my full frame Sony A7R IVA set to crop mode (APS-C). I guess that I can do a one-off edit of the DB to add my model and the crop mode sensor width, but it would be nice to have the option of uncropped and cropped sensor widths, since the make and model does not change in crop mode.

Or, do things work fine when CamerInit sees that the effective focal length is different from the actual focal length?

Cheers,
Paul

**Comment by pchambre** [comment]
Actually, I think there may be some other problem here. Is the Metadata shown in the hover on the yellow icon the same as what the software is trying to match for make and model?

In my case, hovering shows Make: SONY Model: FE 24-70mm F2.8 GM II, which is the make of both body and lens, but the model of the lens. I added an entry to the database exactly like this, and still get no match. I also added an entry exactly like what the metadata shows for the make and model of the camera, and that didn't match either.

Any ideas? Thanks!
